src/main/resources/templates/stream/java/AbstractCharStream.template

```diff
@@ -138,8 +137,7 @@ implements CharStream
 #fi
 
         bufpos -= tokenBegin;
-        // https://github.com/phax/ParserGeneratorCC/pull/23
-        // maxNextCharInd = bufpos;
+        maxNextCharInd = bufpos;
```

Comment:
Why did you re-enable those lines? Makes no difference to me...????

Reply:
As I mentioned before: I wonder what I have to do such that the changes to the templates are picked up by the maven build. E.g. I changed these lines, ran `mvn clean verify` and when I look at the generated AbstractCharStream.java, I still see this in the code
```
        // https://github.com/phax/ParserGeneratorCC/pull/23
        // maxNextCharInd = bufpos;
```

Reply:
When I change the generated code manually, the tests pass.

Reply:
Ah that's what you mean :)
This is because ParserGeneratorCC is build using itself and such changes in this template only effect the "next" version of it.

Reply:
Can you confirm that the test is broken with the current templates but fixed with the reverted change from #23  ?